packages/reactive-element/src/test/decorators/property_test.ts

```diff
@@ -128,15 +128,13 @@ suite('@property', () => {
       _foo?: number;
       updatedContent?: number;
 
-      @property({reflect: true, type: Number})
```

Comment:
How does this code behave with this change? Is it a breaking change?

Reply:
Er, I should say _before_ this change

Reply:
It's not a breaking change with `experimentalDecorators: true` because those read the entire property descriptor off the class and it doesn't matter which is decorated.

Standard decorator usage requires that we decorate the setter though, because we're only passed the setter method.

I can test both in legacy tests.

Reply:
Added a test for decorating a getter in experimental decorators. I don't think we can detect which accessor was decorated, so I couldn't add a deprecation warning.